ndk/lib/src/test/java/io/sentry/ndk/NdkOptionsTest.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry.ndk;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class NdkOptionsTest {
+
+  @Test
+  public void tracesSampleRate() {
```

Comment:
maybe add one more test which verifies that the default value for `options.getTracesSampleRate()` is `0.0`?